### ISSUE TITLE
bower

### ISSUE THREAD

**Issue by hubertrogers** [mbostock/queue#40]
It would be nice to have this registered on bower, so it can be added to projects quickly and easily!

**Comment by mbostock** [comment]
Remind me how I do that?

**Comment by hubertrogers** [comment]
bower init
bower register

http://bower.io/docs/creating-packages/#register

**Comment by mbostock** [comment]
Sort of. I tried:

```
bower init
bower register queue-async git://github.com/mbostock/queue.git
```

It failed:

```
bower                       EDUPLICATE Duplicate package
```

After some searching, it appears that someone has already registered the package under a different name, `asynqueue`, which according to [the registry](http://bower.io/search/?q=async) points to the fork [featness/queue](https://github.com/featness/queue). However, when I installed it, the code came from [heynemann/queue](https://github.com/heynemann/queue) (which seems to redirect to the other, so I assume that user was renamed by GitHub).

Anyway. Then I tried `bower install queue-async`, and it worked.

```
$ bower install queue-async
bower queue-async#*             cached git://github.com/mbostock/queue.git#1.0.7
bower queue-async#*           validate 1.0.7 against git://github.com/mbostock/queue.git#*
bower queue-async#~1.0.7       install queue-async#1.0.7

queue-async#1.0.7 bower_components/queue-async
```

So, either I already registered this component for bower a long time ago and forgot, and the registry search is busted. Or it actually succeeded.  ¯\_(ツ)_/¯
